junction/proposals/models.py

```diff
@@ -144,6 +156,22 @@ class Meta:
 
 
 @python_2_unicode_compatible
+class ProposalSectionReviewerVote(TimeAuditModel):
+
+    """ Reviewer vote for a specific proposal """
+    proposal = models.ForeignKey(Proposal)
+    voter = models.ForeignKey(ProposalSectionReviewer)
+    role = models.PositiveSmallIntegerField(choices=PROPOSAL_USER_VOTE_ROLES, default=2)
+    up_vote = models.BooleanField(default=True)
```

Comment:
This should be `vote` which can accept positive and negative value. Also change in all other places in the pull request. We need to come up with screen transition, indication for reviewer about the vote, ability to change. I will add more comments for ui/ux.